apps/web/src/app/(public)/sign-in/_components/wizard/context.tsx

```diff
@@ -58,10 +60,17 @@ export const SignInWizardProvider = ({
 
   const challengeToVerifierMap = useRef(new Map<string, string>())
   const newChallenge = async () => {
-    const verifier = browserCreatePkceVerifier()
-    const challenge = await browserCreatePkceChallenge(verifier)
-    challengeToVerifierMap.current.set(challenge, verifier)
-    return challenge
+    try {
+      const verifier = browserCreatePkceVerifier()
+      const challenge = await browserCreatePkceChallenge(verifier)
+      challengeToVerifierMap.current.set(challenge, verifier)
+      return challenge
+    } catch (error) {
+      toast.error(
+        'Something went wrong generating a sign-in challenge. Please try again.',
+      )
+      throw error
```

Comment:
this might still bubble up to somewhere, wonder if we should swallow it and do nothing tbh